src/js/franklin-dashboard.js

```diff
@@ -1,81 +1,95 @@
 'use strict';
 
+/* Dependencies */
 import angular from "angular";
 import packageJson from "../../package.json";
-import {
-    LoginComponent
-}
-from "./login/login.controller";
-
 require('angular-ui-router');
```

Comment:
Hmm, I'm using ES6 imports for third party stuff on another project. I'll pull it down see if I can figure out what's happening